### PR TITLE
Bug/pipelines

### PR DESCRIPTION
**Pull request by shapirov103**
*Issue #, if available:* #94

*Description of changes:* Addresses the pipelines issue

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/generic-cluster-construct/index.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ export default class GenericClusterConstruct {
                 new blueprints.MetricsServerAddOn,
                 new blueprints.ClusterAutoScalerAddOn,
                 new blueprints.CloudWatchAdotAddOn,
-                new blueprints.XrayAddOn,
+                new blueprints.XrayAdotAddOn,
```
Comment (elamaran11) [inline lib/generic-cluster-construct/index.ts:63]: jc, why are we removing `XrayAddOn`. I would assume the addon to still work as is.

**Review by elamaran11 — COMMENTED** [review]
Minor comment. There are few files like `lib/aws-batch-on-eks-construct/index.ts` showing link warnings of missing `:` please fix tjose.

**Comment by shapirov103** [comment]
> Minor comment. There are few files like `lib/aws-batch-on-eks-construct/index.ts`, `lib/dynatrace-construct/index.ts`, `lib/datadog-construct/index.ts`, `lib/custom-networking-ipv4-construct/index.ts`, `lib/bottlerocket-construct/index.ts` showing link warnings of missing `:` please fix those.

@elamaran11 The lint warnings are because I added new rules for lint. There are now 54 warnings but I didn't want to clutter this pr with formatting. This will be a separate PR.

File: lib/generic-cluster-construct/index.ts (hunk 1 above)
Comment (shapirov103) [reply lib/generic-cluster-construct/index.ts:63]: It is not compatible with 1.25 and above and I don't see an official way to deploy it at the moment. It is marked as deprecated in the docs. I will deprecate it in the code as well.

Comment (elamaran11) [reply lib/generic-cluster-construct/index.ts:63]: Cool

**Review by elamaran11 — APPROVED** [review]
LGTM

**Pull request merged**